ethportal-peertest/src/lib.rs

```diff
@@ -76,7 +75,28 @@ impl Peertest {
     }
 }
 
-pub async fn launch_node(id: u16, bootnode_enr: Option<&SszEnr>) -> anyhow::Result<PeertestNode> {
+pub async fn launch_node(trin_config: TrinConfig) -> anyhow::Result<PeertestNode> {
+    let web3_ipc_path = trin_config.web3_ipc_path.clone();
+    let server = setup_mock_trusted_http_server();
+    let mock_trusted_provider = TrustedProvider {
+        http: ureq::post(&server.url("/")),
+        ws: None,
+    };
+    let exiter = trin::run_trin(trin_config, mock_trusted_provider)
+        .await
+        .unwrap();
```

Comment:
The short answer is that for some reason, panic handling is broken (especially on Windows?). So every `unwrap()` is a chance for the `win-build` to fail, and fail silently.

In order to [find and fix the last one of these](https://github.com/ethereum/trin/pull/411/commits/cec625ab242ea803a141e4a0db08017824b6fd68), I inserted `error!()` logs before every unwrap, to find the panic. It seems like a good place to start with that is any `unwrap`/`expect`/etc that is new, or moved in this PR.